R/JoinKeys.R

```diff
@@ -331,13 +409,14 @@ mutate_join_keys.TealData <- function(x, dataset_1, dataset_2, val) { # nolint
 #' with relationship to columns of `dataset_2` given by the elements in `keys`.
 #' If `names(keys)` is `NULL` then the same column names are used for both `dataset_1`
 #' and `dataset_2`.
+#' @param parent (`character`) parent dataset
 #'
 #' @return object of class `JoinKeySet` to be passed into `join_keys` function.
 #'
 #' @seealso [join_keys()]
 #'
 #' @export
-join_key <- function(dataset_1, dataset_2, keys) {
+join_key <- function(dataset_1, dataset_2, keys, parent = character(0)) {
```

Comment:
I don't think we should have parent here as it's not a function of the relationship between dataset_1 and dataset_2

Reply:
fair point, it is not used actually as we retrieve the `parents` from the `TealDataset` or `TealDatasetConnector` objects directly.